_pages/en/mandate.md

```diff
@@ -8,40 +8,52 @@ sections: Mission
 permalink: /mandate.html
 ---
 
-## Vision (image of the future)
+- Diagnose key organizational problems, waste, lead times and governance structures that inhibit the desired changes in behaviours
+- Recommend specific course of actions to governance bodies to adopt new methods of managing IT risks
+- Measure progress of adoption of strategies in IT teams and governance bodies as a performance indicator of the IT Strategy team
+- Challenge current thinking and approaches
 
-The [Canadian Experience](human-development-life-cycle.html) is when Canadians are benefiting seamlessly of the federal government services through modern technologies.
+<img src="assets/images/our-values.png" alt="">
+- Learning and Continuous Improvement
+- Self-Discipline
+- Innovation
+- Collaboration
+- Communication
+- Teamwork
 
```

Comment:
Also curious why 'Working in the open' isn't on our values list?  I'm thinking about...when a new team member joins our team, what would we tell them are the values of the team?

Reply:
Missed this one. Working in the open is one of our values.